pkg/limayaml/validate.go

```diff
@@ -523,4 +523,7 @@ func warnExperimental(y *LimaYAML) {
 	if y.MountInotify != nil && *y.MountInotify {
 		logrus.Warn("`mountInotify` is experimental")
 	}
+	if y.SaveOnStop != nil && *y.SaveOnStop {
+		logrus.Warn("`saveOnStop` is experimental")
+	}
```

Comment:
This makes sense but we should add it to the commit adding the option. Do yo plan to squash all the commits to one commit later?